src/config.rs

```diff
@@ -10,6 +10,39 @@ use toml;
 
 type Commands = BTreeMap<String, String>;
 
+#[derive(Debug, PartialEq)]
```

Comment:
I think this can derive `Copy` and it will prevent the need to pass it as reference in `main.rs`

Reply:
Doesn't look like that is the case: https://doc.rust-lang.org/src/core/slice/mod.rs.html#1283-1287

I've tried various combinations of `Copy` and `Clone` with no luck.

Reply:
That being said, we still need it to be at least `Clone` for the new `FromStr` implementation.